lib/seattleflu/db/cli/command/etl/__init__.py

```diff
@@ -243,6 +243,13 @@ def find_sample(db: DatabaseSession, identifier: str) -> Any:
     return sample
 
 
+class SampleNotFoundError(ValueError):
+    """
+    Raised when :func:``update_sample`` is unable to find an existing
```

Comment:
Might want to change doc string? There is an `update_sample` function within `__init__.py` that is does not raise this error

Reply:
Good catch